Update 2.4.2 Turkish Translation

serkan-maker opened the pull request:
# Pull Requests

If you are submitting a pull request, please read https://wiki.audacityteam.org/wiki/GitHub_Pull_Requests

bitigchi commented:
Hi, thanks for the translations. I'll continue the review later on, but so far these translations are hardly an improvement over the current ones, moreover some of the previously correct translations are broken. I'd recommend a re-pass over the translations as a whole, and make sure that the "audio" context is preserved.

serkan-maker commented:
Hello, I corrected some incorrect translations. I did not make much change in the current translation. The majority was to add accelerators. I fixed the translation of the most parts. I would appreciate it if you report the mistake I made, I would be considered a novice in the translation business. There are also 270 missing translation lines that are not in the .pot file but are available in the interface. These missing lines are present in the .po file but cannot be used.
[audacity_tr.po.zip](https://github.com/audacity/audacity/files/6161718/audacity_tr.po.zip)

bitigchi commented on locale/tr.po:
```diff
@@ -20893,121 +20240,107 @@ msgid ""
+#, lisp-format
 msgid "Error.~%Unable to open file"
-msgstr "Tür dosyası açılamadı."
+msgstr "Hata. ~% Dosya açılamıyor"
```
yüzde işaretinden sonra boşluk yok

bitigchi commented on locale/tr.po:
```diff
@@ -20239,551 +19623,512 @@ msgid ""
 #, lisp-format
 msgid "<b>DC Offset:</b> &nbsp;&nbsp;~a"
-msgstr ""
+msgstr "<b>DC Offset:</b> &nbsp;&nbsp;~a"
```
Ofseti

bitigchi commented on locale/tr.po:
```diff
@@ -20239,551 +19623,512 @@ msgid ""
 #, lisp-format
 msgid "Left: ~a lin, ~a dB | Right: ~a lin, ~a dB."
-msgstr ""
+msgstr "Sol: ~a lin, ~a dB | Sağ: ~a lin, ~a dB."
```
Lin yerine doğ veya doğrusal

bitigchi commented on locale/tr.po:
```diff
@@ -20239,551 +19623,512 @@ msgid ""
 #, lisp-format
 msgid "~aData written to:~%~a"
-msgstr ""
+msgstr "~aVeri yazılan:~%~a"
```
Veri şuraya yazıldı:

bitigchi commented on locale/tr.po:
```diff
@@ -20239,551 +19623,512 @@ msgid ""
-#, fuzzy
 msgid "Export data to"
-msgstr "Sesi Dışa Aktar"
+msgstr "Verileri şuraya aktar"
```
: eksik

bitigchi commented on locale/tr.po:
```diff
@@ -20239,551 +19623,512 @@ msgid ""
 #: plug-ins/rissetdrum.ny
 msgid "Risset Drum"
-msgstr ""
+msgstr "Risset Davul"
```
Davulu. Altta da aynı terim için tambur denmiş.

bitigchi commented on locale/tr.po:
```diff
@@ -20239,551 +19623,512 @@ msgid ""
 "'Rhythm track duration' to greater than zero."
 msgstr ""
+"'Çubuk sayısı' nı veya\n"
+"'Ritim izleme süresi' sıfırdan büyük."
```
tırnak işaretinden sonra ek gelirse boşlukla ayrılmaz

bitigchi commented on locale/tr.po:
```diff
@@ -20239,551 +19623,512 @@ msgid ""
 #: plug-ins/rhythmtrack.ny
 msgid "MIDI pitch of weak beat"
-msgstr ""
+msgstr "MIDI zayıf vuruş aralığı"
```
Zayıf vuruşun MIDI perdesi

bitigchi commented on locale/tr.po:
```diff
@@ -20239,551 +19623,512 @@ msgid ""
-#, fuzzy
 msgid "Rhythm track duration"
-msgstr "İzi Kaldır"
+msgstr "Ritim parça süresi"
```
bazı yerlerde parça, bazı yerlerde iz kullanılmış. hangisi?

bitigchi commented on locale/tr.po:
```diff
@@ -20239,551 +19623,512 @@ msgid ""
 #: plug-ins/rhythmtrack.ny
 msgid "1 - 1000 bars"
-msgstr ""
+msgstr "1 - 1000 bars"
```
aşağıda çubuk varken çoğu yerde bar olarak çevrilmeden bırakılmış

bitigchi commented on locale/tr.po:
```diff
@@ -20199,38 +19581,40 @@ msgid ""
-#, fuzzy
 msgid "Notch Filter"
-msgstr "Süzgeç Uzunluğu"
+msgstr "Notch Filtresi"
```
süzgeç varken neden filtre? aynı şey ve biri Türkçe

bitigchi commented on locale/tr.po:
```diff
@@ -20085,95 +19471,84 @@ msgid ""
-#, fuzzy
 msgid "Don't Link Stereo"
-msgstr "Ekli Çift Kanallı"
+msgstr "Stereo suz"
```
-sXz eki neden ayrı? çift kanallı varken neden stereo?

bitigchi commented on locale/tr.po:
```diff
@@ -19932,151 +19330,139 @@ msgid ""
+#, lisp-format
 msgid "Error.~%Selection must be less than ~a."
-msgstr "Seçimde en az %d örnek bulunmalı."
+msgstr "Hata.~%Seçim, ~a'dan küçük olmalıdır."
```
çevirirken değişkenlere kesme işaretiyle ek getirilmez, değişkenin büyük ünlü uyumuna uyacağı garantisi yok.

bitigchi commented on locale/tr.po:
```diff
@@ -19932,151 +19330,139 @@ msgid ""
-#, fuzzy
 msgid "RMS level"
-msgstr "Islak düzey"
+msgstr "RMS seviyesi"
```
düzeyin nesi yanlış?

bitigchi commented on locale/tr.po:
```diff
@@ -19506,424 +18951,377 @@ msgid ""
 #: plug-ins/beat.ny
 msgid "Finding beats..."
-msgstr ""
+msgstr "Vuruş bulmak..."
```
... varsa süren bir eylemi belirtir. "Vuruşlar bulunuyor..."

bitigchi commented on locale/tr.po:
```diff
@@ -19506,424 +18951,377 @@ msgid ""
-#, fuzzy
 msgid "Beat Finder"
-msgstr "&Vuruş"
+msgstr "Ritim Bulucu"
```
vuruş varken ritim ne alaka? ritim rhythm yerine kullanılıyor zaten

bitigchi commented on locale/tr.po:
```diff
@@ -19329,175 +18795,154 @@ msgid ""
+#, lisp-format
 msgid "~aPercentage values cannot be more than 1000 %."
-msgstr "Frekans kazancı negatif olamaz"
+msgstr "~aYüzde değerleri % 1000'den fazla olamaz."
```
% burada yüzde işaretini değil de bir değişkeni belirtiyor olsa gerek. başa yazılması hatalı.

bitigchi commented on locale/tr.po:
```diff
@@ -19329,175 +18795,154 @@ msgid ""
-#, fuzzy
 msgid "Start/End as"
-msgstr "Başlangıç - Bitiş"
+msgstr "Başla/Bitir"
```
sonunda as olduğu için başka bir ifade izliyor olsa gerek. Şunun gibi başla/bitir: daha yerinde olabilir. Programdan kontrol edilmeli.

bitigchi commented on locale/tr.po:
```diff
@@ -19329,175 +18795,154 @@ msgid ""
-#, fuzzy
 msgid "Fade Up"
-msgstr "Artarak Gir"
+msgstr "Karart"
```
Karartma anlaşılır değil. Sesi açarak gir, diğeri için sesi azaltarak çık vb. gibi terimler kullanılmalı.

bitigchi commented on locale/tr.po:
```diff
@@ -19329,175 +18795,154 @@ msgid ""
-#, fuzzy
 msgid "Applying Fade..."
-msgstr "Uygulanıyor..."
+msgstr "Solma Uygulanıyor..."
```
solma? karartma? hangisi?

bitigchi replied:
aşağıda belirttiğim gibi açma/azaltma en iyi karşılık fade için duruma göre

bitigchi commented on locale/tr.po:
```diff
@@ -19216,55 +18679,51 @@ msgid "File Dialog"
+#: plug-ins/vocalrediso.ny plug-ins/vocoder.ny
 msgid "Released under terms of the GNU General Public License version 2"
-msgstr ""
+msgstr "GNU Genel Kamu Lisansı sürüm 2'nin koşulları altında yayınlandı"
```
GNU Genel Kamu Lisansı, ikinci sürümün koşulları altında yayımlanmıştır.

bitigchi commented on locale/tr.po:
```diff
@@ -19051,9 +18517,8 @@ msgstr "100>01000 oktav|1.442695041"
-#, fuzzy
 msgid "thousandths of octaves"
-msgstr "binde "
+msgstr "binde octave"
```
binlerce oktav

bitigchi commented on locale/tr.po:
```diff
@@ -19010,13 +18477,12 @@ msgstr "01000,01000 kare|75"
 #: src/widgets/NumericTextCtrl.cpp
 msgid "centihertz"
-msgstr ""
+msgstr "centihertz"
```
santihertz

bitigchi commented on locale/tr.po:
```diff
@@ -18478,45 +17952,40 @@ msgstr "Parçalar başka bir ize taşındı"
-#, fuzzy
 msgid "Command+Click"
-msgstr "Command-Tıklayın"
+msgstr "Komut-Tıklayın"
```
Command düğmesi olduğu gibi bırakılmalı, klavyelerin üzerinden aynen böyle yazıyor.

bitigchi commented on locale/tr.po:
```diff
@@ -18147,7 +17629,7 @@ msgstr "Kanalları Değiştir"
 #, c-format
 msgid "Split stereo track '%s'"
-msgstr "'%s' çift kanallı izini ayır"
+msgstr "'%s' stereo parçasını böl"
```
çift kanallı'nın nesi yanlış?

bitigchi commented on locale/tr.po:
```diff
@@ -18087,21 +17570,20 @@ msgstr "Oranı Ayarla"
-#, fuzzy
 msgid "&Multi-view"
-msgstr "Çoklu"
+msgstr "&Çoklu-görünüm"
```
bu bağlamda Türkçede tire kullanılmaz, böyle bir kural yok. "Çoklu görünüm"

bitigchi commented on locale/tr.po:
```diff
@@ -17989,19 +17473,18 @@ msgid "&Format"
-#, fuzzy
 msgid "Changing sample format"
-msgstr "Örnekleme hızı geçersiz"
+msgstr "Örnek formatın değiştirilmesi"
```
format kullanılmaya başlansa da biçim hala çok yaygın çevirilerde. değiştirilmesi için bir neden yok

bitigchi replied:
artı çeviri hatalı. "Örnek biçim değiştiriliyor"

bitigchi commented on locale/tr.po:
```diff
@@ -17956,25 +17442,23 @@ msgstr "Örnek Düzenleme"
-"Spektrogram ayarlarını değiştirmek için önce tüm\n"
-"oynatma ya da kaydetme işlemlerini durdurun."
+"Spektrogram Ayarlarını değiştirmek için herhangi bir\n"
+" önce oynatma veya kaydetme."
```
doğru olan çeviriyi neden değiştiriyorsunuz?

bitigchi commented on locale/tr.po:
```diff
@@ -17956,25 +17442,23 @@ msgstr "Örnek Düzenleme"
 #: src/tracks/playabletrack/wavetrack/ui/SpectrumVRulerControls.cpp
 msgid "k"
-msgstr ""
+msgstr "k"
```
" bin" olarak çevrilmeli (boşluğa dikkat). Türkçede 10k diye bir kullanım yok.

bitigchi commented on locale/tr.po:
```diff
@@ -17879,16 +17367,14 @@ msgstr "Yakınlaştırmayı Sıfırla"
-#, fuzzy
 msgid "Shift-Right-Click"
-msgstr "Shift ile Sol Tıklama"
+msgstr "Shift-Sağ-Tık"
```
yanlış tire kullanımı...

bitigchi commented on locale/tr.po:
```diff
@@ -17707,9 +17201,8 @@ msgid "Draw Tool"
-#, fuzzy
 msgid "Multi-Tool"
-msgstr "Çoklu Araç"
+msgstr "Çoklu-Araç"
```
yanlış tire kullanımı

bitigchi commented on locale/tr.po:
```diff
@@ -17343,11 +16841,11 @@ msgstr "Kayıt Kanalları"
 #: src/toolbars/DeviceToolBar.cpp
 msgid "2 (Stereo) Recording Channels"
-msgstr "2 (Stereo) Kayıt Kanalı"
+msgstr "2 (Stereo) Kayıt"
```
kanal neden silindi?

bitigchi commented on locale/tr.po:
```diff
@@ -17178,29 +16681,28 @@ msgstr "&Kafa sabitlenmemiş ise otomatik kaydırılsın"
 #: src/prefs/TracksPrefs.cpp
 msgid "Audio Track"
-msgstr "Ses İzi"
+msgstr "Müzik parçası"
```
müzik ne alaka?

bitigchi commented:
I will continue reviewing later on. It will take a while, since I was able to barely review a quarter of the changes.

serkan-maker commented:
I corrected the wrong translations you warned
[tr.po.zip](https://github.com/audacity/audacity/files/6163845/tr.po.zip)

bitigchi commented:
> I corrected the wrong translations you warned
> [tr.po.zip](https://github.com/audacity/audacity/files/6163845/tr.po.zip)

Thanks for the corrections, I appreciate it. Is it possible to commit your changes here for easier review?

bitigchi commented on locale/tr.po:
```diff
@@ -722,17 +712,17 @@ msgstr "Gönderim Kodu:"
+#, c-format
 msgid "Release build (debug level %d)"
-msgstr "Yayın sürümü"
+msgstr "Sürüm derlemesi (hata ayıklama düzeyi %d)"
```
Yayın sürümü/yapısı daha doğru. Sürüm derlemesi anlam bakımından doğru; ancak kavramlar karışıyor.

bitigchi commented on locale/tr.po:
```diff
@@ -819,14 +804,12 @@ msgid "Move to Scrub"
-#, fuzzy
 msgid "Drag to Seek. Release and move to Scrub."
-msgstr "Taramak için sürükleyin. Sarmak için bırakıp taşıyın."
+msgstr "Aramak için sürükleyin. Bırakın ve Scrub'a gidin."
```
Yeni çeviri hatalı, eskisi doğru. Scrub'a gidin bir anlam ifade etmiyor. Aramak da sözlük anlamı olarak doğru olsa da kavram olarak hatalı.

bitigchi commented on locale/tr.po:
```diff
@@ -1003,12 +984,11 @@ msgid ""
 #: src/AudacityApp.cpp
 msgid "Do you still want to start Audacity?"
-msgstr "Audacity gene de başlatılsın mı?"
+msgstr "Hala Audacity'yi başlatmak istiyor musunuz?"
```
Audacity'yi yine de başlatmak istiyor musunuz?

Çevirirken kavramları çevirmeye sondan başlamak daha doğru. Özne burada Audacity.

bitigchi commented on locale/tr.po:
```diff
@@ -1140,30 +1137,41 @@ msgid ""
-#, fuzzy
 msgid "&Quit Audacity"
-msgstr "Audacity'den Çık"
+msgstr "Audacity'den &Çık"
```
Türkçe karaktere atanan kısayol

bitigchi commented on locale/tr.po:
```diff
@@ -1558,99 +1552,87 @@ msgstr "&Komut"
 #: src/BatchCommandDialog.cpp
 msgid "&Use Preset"
-msgstr "Hazır Ayar K&ullanılsın"
+msgstr "Ön Ayarı &Kullan"
```
Yeni çeviri hatalı, eskisi doğru

bitigchi commented on locale/tr.po:
```diff
@@ -1558,99 +1552,87 @@ msgstr "&Komut"
-#, fuzzy
 msgid "Fade Ends"
-msgstr "Artarak Gir"
+msgstr "Bitişte Ses seviyesini azalt"
```
Tutarsız büyük harf kullanımı

bitigchi commented on locale/tr.po:
```diff
@@ -1558,99 +1552,87 @@ msgstr "&Komut"
-#, fuzzy
 msgid "Import Macro"
-msgstr "Ham Veri İçe Aktarma"
+msgstr "Makroyu İçe Aktar"
```
-yu gereksiz

bitigchi commented on locale/tr.po:
```diff
@@ -1558,99 +1552,87 @@ msgstr "&Komut"
-#, fuzzy
 msgid "Export Macro"
-msgstr "MIDI Dışa Aktar"
+msgstr "Makroyu Dışa Aktar"
```
-yu gereksiz

bitigchi commented on locale/tr.po:
```diff
@@ -1558,99 +1552,87 @@ msgstr "&Komut"
 #: src/BatchCommands.cpp
 msgid "Menu Command (No Parameters)"
-msgstr ""
+msgstr "Menü Komutu (Parametre Yok)"
```
Parametresiz

bitigchi commented on locale/tr.po:
```diff
@@ -1558,99 +1552,87 @@ msgstr "&Komut"
 #, c-format
 msgid "Your batch command of %s was not recognized."
-msgstr "%s yığın komutunuz anlaşılamadı."
+msgstr "%s toplu komutunuz tanınmadı."
```
anlaşılamadı bu bağlamda doğru. batch için toplu iş olarak kullanılır genelde. tanınmadı doğrudan çeviri oluyor.

bitigchi commented on locale/tr.po:
```diff
@@ -1751,43 +1726,40 @@ msgstr "Dosya"
 #: src/BatchProcessDialog.cpp
 msgid "&Cancel"
-msgstr "İ&ptal"
+msgstr "&İptal"
```
türkçe karakterde kısayol

bitigchi commented on locale/tr.po:
```diff
@@ -1751,43 +1726,40 @@ msgstr "Dosya"
-#, fuzzy
 msgid "Remo&ve"
-msgstr "Kaldır"
+msgstr "Kald&ır"
```
Türkçe karakterde kısayol

bitigchi commented on locale/tr.po:
```diff
@@ -1751,43 +1726,40 @@ msgstr "Dosya"
 #: src/BatchProcessDialog.cpp src/LabelDialog.cpp src/effects/Equalization.cpp
 msgid "I&mport..."
-msgstr "&Al..."
+msgstr "İ&çe aktar..."
```
Türkçe karakterde kısayol

bitigchi commented on locale/tr.po:
```diff
@@ -1751,43 +1726,40 @@ msgstr "Dosya"
+#: src/BatchProcessDialog.cpp src/effects/Contrast.cpp src/effects/Equalization.cpp
 msgid "E&xport..."
-msgstr "V&er..."
+msgstr "D&ışa aktar..."
```
Türkçe karakterde kısayol

serkan-maker commented:
incorrect translation corrections
[tr.po.zip](https://github.com/audacity/audacity/files/6169563/tr.po.zip)
